functions/exercise23/script.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+  function findFirstNotRepeatedCharacter(phrase) {
+      let firstNotRepeatedCharacter = "";
+      for(let i = 0; i < phrase.length; i++){
+          if(phrase.indexOf(phrase[i]) === phrase.lastIndexOf(phrase[i])){
```

Comment:
You refer to phrase[i] 3 times. Can't you put it into local variable and refer to this local variable?